protocol_details/system_considerations.rst

```diff
@@ -2,41 +2,80 @@
 System Wide Considerations
 ==========================
 
-AMP systems could either be supervised (using a hypervisor to enforce isolation and resource virtualization) or unsupervised (modifying each participating software context to ensure best- effort isolation and cooperative usage of shared resources). With unsupervised AMP systems, there is no strict isolation or supervision of shared resource usage.
+AMP systems could either be supervised (using a hypervisor to enforce isolation and resource
+virtualization) or unsupervised (modifying each participating software context to ensure best-
+effort isolation and cooperative usage of shared resources). With unsupervised AMP systems, there is
+no strict isolation or supervision of shared resource usage.
 
-Take the following system-wide considerations into account to develop unsupervised AMP systems using the OpenAMP framework.
+Take the following system-wide considerations into account to develop unsupervised AMP systems
+using the OpenAMP framework.
 
    - Determine system :ref:`architecture/topology<topology-work-label>`
 
-   The OpenAMP framework implicitly assumes a host controller to remote system architecture. The topology for this architecture should be determined; either star, chain, or a combination. The following figure shows some simple use cases.
+   The OpenAMP framework implicitly assumes a host controller to remote system architecture. The
+   topology for this architecture should be determined; either star, chain, or a combination. The
+   following figure shows some simple use cases.
 
-      * Case 1 — A single host controller software context on processor 1 controlling life cycle and communicating with two independent remote software contexts on processors 2 and 3, in star topology,
+      * Case 1 — A single host controller software context on processor 1 controlling life cycle and
+        communicating with two independent remote software contexts on processors 2 and 3, in star
+        topology,
 
-      * Case 2 — Host controller software context 1 on processor 1 brings up remote software context 1 on processor 2. This context acts as the host software context 2 for remote software context 2 on processor 3, in chain topology.
+      * Case 2 — Host controller software context 1 on processor 1 brings up remote software
+        context 1 on processor 2. This context acts as the host software context 2 for remote
+        software context 2 on processor 3, in chain topology.
 
    .. image:: ../images/topo_types.svg
```

Comment:
The topo_types image is not visible [here](https://openamp--54.org.readthedocs.build/en/54/protocol_details/system_considerations.html#system-wide-considerations).

![image](https://github.com/user-attachments/assets/c3aa69dd-fd0b-4065-b952-7b31e810eb94)

Reply:
done

Reply:
@arnopo protocol_details/system_considerations.rst still includes the incorrectly names topo_types.svg